### PR TITLE
Dashboard view

### PR DESCRIPTION
**Pull request by jasonmadigan**
Draft PR, implements #33 

Add's Kuadrant dashboard view. 

TODID:

- [x] Gateways list
- [x] API / HTTPRoute list
- [x] Alerts list?
- [x] i18n

Screenshot:
Screenshot 2024-09-17 at 16 56 50

TODO later:

- [ ] Final links (this will have to wait have to wait), will create a follow-up issue
- [ ] Tweaks to status/ conditions in lists (will come later)

Out of scope for now: Alerts

### CODE REVIEW

File: src/constants/links.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+// Internal Links
+export const INTERNAL_LINKS = {
+  createPolicies: "/kuadrant/all-namespaces/policies",
+  apiDesignerOperator: (namespace: string) =>
```
Comment (jasonmadigan) [inline src/constants/links.ts:4]: This is TBD IMHO - I don't think we'll have an operator for some time, likely a docs link

File: src/constants/links.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+// External Links
+export const EXTERNAL_LINKS = {
+  // TODO: Update these when available for real
+  documentation: "https://access.redhat.com/articles/7065949",
+  releaseNotes: "https://access.redhat.com/articles/7065949",
+  quickStarts: "https://access.redhat.com/articles/7065949",
+  highlights: "#", 
+  blog: "https://access.redhat.com/articles/7065949",
+};
```
Comment (jasonmadigan) [inline src/constants/links.ts:21]: these should be upstream links

Comment (jasonmadigan) [reply src/constants/links.ts:20]: resolved

**Comment by david-martin** [comment]
👀

**Comment by jasonmadigan** [comment]
Made a slight tweak just now to make the getting started resources section: 
- collapsible the arrow
- hideable for a session via an action

File: src/constants/links.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  certManagerOperator: (namespace: string) =>
+    `/operatorhub/ns/${namespace === '#ALL_NS#' ? 'default' : namespace}?keyword=cert-manager&details-item=openshift-cert-manager-operator-redhat-operators-openshift-marketplace`
+};
+
+// External Links
+export const EXTERNAL_LINKS = {
+  // TODO: Update these when available for real
+  documentation: "https://docs.kuadrant.io",
+  releaseNotes: "https://docs.kuadrant.io/0.8.0/",
+  quickStarts: "https://docs.kuadrant.io/0.8.0/kuadrant-operator/doc/user-guides/secure-protect-connect/",
```
Comment (david-martin) [inline src/constants/links.ts:17]: can we change to /latest ?

File: src/constants/links.ts [hunk 4]
```diff
@@ -0,0 +1,20 @@
+// Internal Links
+export const INTERNAL_LINKS = {
+  createPolicies: "/kuadrant/all-namespaces/policies",
+  apiDesigner: 'https://www.apicur.io/studio/',
+  addNewGateway: (namespace: string) =>
+    `/k8s/ns/${namespace === '#ALL_NS#' ? 'default' : namespace}/gateway.networking.k8s.io~v1~Gateway/~new`,
+  observabilitySetup: 'https://docs.kuadrant.io/0.10.0/kuadrant-operator/doc/observability/examples/',
```
Comment (david-martin) [inline src/constants/links.ts:7]: can we change to /latest ?

File: src/constants/links.ts [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+  certManagerOperator: (namespace: string) =>
+    `/operatorhub/ns/${namespace === '#ALL_NS#' ? 'default' : namespace}?keyword=cert-manager&details-item=openshift-cert-manager-operator-redhat-operators-openshift-marketplace`
+};
+
+// External Links
+export const EXTERNAL_LINKS = {
+  // TODO: Update these when available for real
+  documentation: "https://docs.kuadrant.io",
+  releaseNotes: "https://docs.kuadrant.io/0.8.0/",
```
Comment (david-martin) [inline src/constants/links.ts:16]:
```suggestion
  releaseNotes: "https://github.com/Kuadrant/kuadrant-operator/releases",
```

File: src/constants/links.ts (hunk 3 above)
Comment (jasonmadigan) [reply src/constants/links.ts:17]: yep

**Review by david-martin — APPROVED** [review]
/lgtm

**Pull request merged**